test/server/cards/03_TWI/units/BoKatanKryzeDeathWatchLieutenant.spec.ts

```diff
@@ -0,0 +1,38 @@
+describe('Bo-Katan Kryze, Death Watch Lieutenant', function () {
+    integration(function (contextRef) {
+        describe('Bo-Katan Kryze\'s ability', function () {
+            it('should get +1/+0 as we control a trooper unit', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['bokatan-kryze#death-watch-lieutenant', 'battlefield-marine'],
+                    },
+                });
+
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.bokatanKryze);
+                expect(context.p2Base.damage).toBe(3);
```

Comment:
Can we have P2 vanquish the Marine and ensure Bo-Katan isn't buffing herself? Ditto with the other effects in the other test